webapp/src/delete/i18n/i18n.jsx

```diff
@@ -0,0 +1,196 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
```

Comment:
I'm assuming this is all to support the extracted component? I'd suggest dropping all the localization and assume we expose the component in some way in the future instead, since there are a lot more strings here than needed, and we wouldn't be able to maintain these translations consistently anyway.

Reply:
Agreed!

Reply:
removed by 5938c68602c6d7452063c377277bd7ad44edde6c

Reply:
removed by 9a871503e422a8d52e2ceb0293728c112714b373